scripts/SetGateVariableTemplateCI.ps1

```diff
@@ -0,0 +1,46 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]
+    $BaseImageName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $stackname,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $VersionTextTemplateCI
```

Comment:
The parameter name should just be $Version

Reply:
> Not able use **$Version** as it already there with small v **$version** in the code and PowerShell considering both same and returning the same value.

PowerShell is case insensitive anyway.

The parameter must be `$Version`.

Change the script variable to another name. `$VersionLocal` maybe.

Reply:
Parameter name renamed to `$Version` and script variable name rename to `$VersionLocal`